### DISCUSSION TITLE
Error in Scenicplus installation

### DISCUSSION THREAD

**Discussion by Umaarasu** [aertslab/scenicplus#106] [Q&A] [answered]
Hi,
I get this error when I try to install scenicplus using conda. In Macs githib page few others have also reported this error while installing scenicplus. In one thread they have closed the report saying install Macs3. But installing Macs3 does not help with installing scenic plus package dependencies which requires Macs2. Could this be fixed?

× Preparing metadata (pyproject.toml) did not run successfully.
│ exit code: 1
╰─> [4 lines of output]
Requirement already satisfied: numpy>=1.17 in /tmp/pip-build-env-880f00gu/overlay/lib/python3.8/site-packages (1.24.2)
error in MACS2 setup command: 'install_requires' must be a string or list of strings containing valid project/version requirement specifiers; Expected end or semicolon (after name and no valid version specifier)
numpy>=>=1.17
^
[end of output]

note: This error originates from a subprocess, and is likely not a problem with pip.
error: metadata-generation-failed

Version :
OS: Linux
Python version 3.8.15

**Comment by SeppeDeWinter** [comment, marked as answer]
See issue: https://github.com/aertslab/scenicplus/issues/101
